packages/core/src/etherscan.ts

```diff
@@ -311,6 +310,12 @@ export const etherscanVerifySphinxSystem = async (
       let buildInfo: BuildInfo
       if (type === SystemContractType.SPHINX) {
         buildInfo = sphinxBuildInfo
+        // We have to remove `.Darwin.appleclang` to make etherscan verify our contracts
+        // This could also be an issue when verifying end user contracts, but we haven't seen it so far
+        buildInfo.solcLongVersion = buildInfo.solcLongVersion.replace(
+          '.Darwin.appleclang',
+          ''
+        )
```

Comment:
See the comment; we should keep an eye out for this being a problem for users deploying via the DevOps platform. So far, I haven't encountered this issue and I've been deploying using the same machine I used to compile our core contracts.